wp-content/plugins/core/src/Blocks/Types/Post_List/Post_List.php

```diff
@@ -0,0 +1,213 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Blocks\Types\Post_List;
+
+use Tribe\Libs\ACF\Field;
+use Tribe\Libs\ACF\Block;
+use Tribe\Libs\ACF\Block_Config;
+use Tribe\Project\Post_Types\Post\Post;
+use Tribe\Project\Post_Types\Sample\Sample;
+use Tribe\Project\Theme\Config\Image_Sizes;
+
+class Post_List extends Block_Config {
+	public const NAME = 'postlist';
+
+	public const LEAD_IN     = 'leadin';
+	public const TITLE       = 'title';
+	public const DESCRIPTION = 'description';
+	public const CTA         = 'cta';
+
+	public const QUERY_TYPE        = 'query_type';
+	public const QUERY_TYPE_AUTO   = 'query_type_auto';
+	public const QUERY_TYPE_MANUAL = 'query_type_manual';
+
+	public const POSTS      = 'posts'; //Conditional to Manual
+	public const LIMIT      = 'limit'; //Conditional to Auto
+	public const TAXONOMIES = 'taxonomy_terms'; //Conditional to Auto, one per post type
+	public const POST_TYPES = 'post_types'; //Conditional to Auto
+
+	public const LAYOUT         = 'layout';
+	public const LAYOUT_INLINE  = 'inline';
+	public const LAYOUT_STACKED = 'stacked';
+
+	public const QUERY_POST_TYPES = [
+		Post::NAME,
+		Sample::NAME,
+	];
+
+	/**
+	 * Register the block
+	 */
+	public function add_block() {
+		$this->set_block( new Block( self::NAME, [
+			'title'       => __( 'Post List', 'tribe' ),
+			'description' => __( 'A block of curated posts', 'tribe' ),
+			'icon'        => 'sticky',
+			'keywords'    => [ __( 'posts', 'tribe' ), __( 'display', 'tribe' ), __( 'text', 'tribe' ) ],
+			'category'    => 'layout',
+			'supports'    => [ 'align' => false ],
+		] ) );
+	}
+
+	/**
+	 * Register Fields for block
+	 */
+	public function add_fields() {
+		$this->add_field( new Field( self::NAME . '_' . self::TITLE, [
+				'label' => __( 'Title', 'tribe' ),
+				'name'  => self::TITLE,
+				'type'  => 'text',
+			] )
+		)->add_field( new Field( self::NAME . '_' . self::LEAD_IN, [
+				'label' => __( 'Lead in', 'tribe' ),
+				'name'  => self::LEAD_IN,
+				'type'  => 'text',
+			] )
+		)->add_field( new Field( self::NAME . '_' . self::DESCRIPTION, [
+				'label'        => __( 'Description', 'tribe' ),
+				'name'         => self::DESCRIPTION,
+				'type'         => 'wysiwyg',
+				'toolbar'      => 'basic',
+				'media_upload' => 0,
+			] )
+		)->add_field( new Field( self::NAME . '_' . self::CTA, [
+				'label' => __( 'Call to Action', 'tribe' ),
+				'name'  => self::CTA,
+				'type'  => 'link',
+			] )
+		)->add_field( new Field( self::NAME . '_' . self::QUERY_TYPE, [
+				'label'   => __( 'Type of Query', 'ur-core' ),
```

Comment:
usage of `ur-core` to `tribe` through out

Reply:
🍝